### ISSUE TITLE
Version 0.27.0, 0.27.1: undefined method `ruby2_keywords' for MockRedis::GeospatialMethods:Module

### ISSUE THREAD

**Issue by tsu-shiuan** [sds/mock_redis#204]
```
Failure/Error: require 'mock_redis'

NoMethodError:
  undefined method `ruby2_keywords' for MockRedis::GeospatialMethods:Module
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis/geospatial_methods.rb:15:in `<module:GeospatialMethods>'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis/geospatial_methods.rb:2:in `<class:MockRedis>'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis/geospatial_methods.rb:1:in `<top (required)>'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis/database.rb:12:in `require'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis/database.rb:12:in `<top (required)>'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis.rb:4:in `require'
# ./bundle/gems/mock_redis-0.27.1/lib/mock_redis.rb:4:in `<top (required)>'
```

Ruby version: 2.6.6

Last working mock_redis version: 0.26.0

Steps to replicate:
```
require 'mock_redis'
```

**Comment by tsu-shiuan** [comment]
I think the fix would be to change `ruby2_keywords` into a runtime dependency in the [gemspec](https://github.com/sds/mock_redis/blob/ca645abdfa24def05944cd86a4d7c9c9e8d29e87/mock_redis.gemspec#L29)👇 

```
s.add_runtime_dependency 'ruby2_keywords'
```

**Comment by sds** [comment]
Released in [0.27.2](https://github.com/sds/mock_redis/releases/tag/v0.27.2).
